src/main/java/com/billingapi/controller/InvoiceController.java

```diff
@@ -19,45 +19,64 @@
 @RequestMapping("/invoice")
 @Validated
 public class InvoiceController {
+    static Logger logger= LogManager.getLogger(InvoiceController.class);
     @Autowired
     private InvoiceService invoiceService;
 
 
-    //To find Invoice
+
+
+    /*  To find Invoice
+       * @param  invoiceId an Id giving the Invoice of specific Id
+       * @return      Invoice of specific Id.
+       * @throws InvoiceNotFoundException  If an Invoice of specific Id is not found
+    */
     @GetMapping("/{invoiceId}")
     public ResponseEntity<?> findInvoiceById(@PathVariable int invoiceId) throws InvoiceNotFoundException {
         Invoice invoice = invoiceService.findInvoiceById(invoiceId);
         if (invoice == null) {
```

Comment:
if you can implement something ..
 return invoiceService.findById(invoiceId).orElseThrow(
        () -> new ResponseStatusException(HttpStatus.NOT_FOUND,
            "Invpice id " + invoiceId+ " was not found"));